packages/editor/templates/Hello World/README.md

```diff
@@ -1,198 +1,95 @@
+Then deploy the contract by selecting the "_Deploy_" option within the Deploy panel.
 
-First create an instance of the smart contract, [passing it as a property](https://web3js.readthedocs.io/en/v1.2.0/web3-eth-contract.html), which allows web3js to interact with it.
+You should now see the deployed contract's `message` variable displayed on the IDE's Browser as well as output from the transaction in the IDE's console (on the lower right side of the IDE).
 
-```javascript
-function HelloWorld(Contract) {
-    this.web3 = null;
-    this.instance = null;
-    this.Contract = Contract;
-}
-```
+### 3. Interact
 
-Initialize the `HelloWorld` object and create an instance of the web3js library, passing Metamask as a provider for the contract. The initialization function then defines the interface for the contract using [the web3js contract object](https://web3js.readthedocs.io/en/v1.2.1/web3-eth-contract.html#new-contract) and then defines the address of the instance of the contract for the `HelloWorld` object.
+Now check out the Interaction panel on the left side of this IDE (the mouse icon).
 
-```javascript
-HelloWorld.prototype.init = function () {
+Here you view and interact with your deployed contract using its functions. Try updating the `message` variable using the `update` function. This will create a new Ethereum transaction and you should see the message update in the IDE's Browser.
 
-    this.web3 = new Web3(
-        (window.web3 && window.web3.currentProvider) ||
-        new Web3.providers.HttpProvider(this.Contract.endpoint));
+## The web app (dapp)
 
-    var contract_interface = this.web3.eth.contract(this.Contract.abi);
+Often when creating an Ethereum smart contract, it's useful to create a web application for users to interact with. We call these applications "dapps". [Dapps on Ethereum](https://ethereum.org/dapps/) are web applications backed by Ethereum smart contracts. Instead of using a centralized server or database, these applications rely on the blockchain as a backend for program logic and storage.
 
-    this.instance = this.Contract.address ? contract_interface.at(this.Contract.address) :  { message: () => {} };
-};
-```
+Dapps typically use a [JavaScript convenience libraries](https://ethereum.org/developers/#frontend-javascript-apis) that provide an API to make integrations with smart contract easier for developers. In this project, you'll be using [web3.js](https://web3js.readthedocs.io/en/v1.2.8/).
 
-Add other JavaScript boilerplate to create the instance of the `HelloWorld` object defined above, show the HTML elements on the page, and bind the functions for interacting with the contract to the button defined in the HTML:
+This tutorial won't cover the HTML or CSS since it's not specific to a dapp, although it's worth noting that this application uses jQuery to manipulate the HTML (of _Files/app/app.html_) that is utlimately rendered in our IDE's Browser.
```

Comment:
```suggestion
This tutorial won't cover the HTML or CSS since it's not specific to a dapp, although it's worth noting that this application uses jQuery to manipulate the HTML (of _Files/app/app.html_) that is ultimately rendered in the IDE's Browser.
```